src/handbook/customer/marketing/customer-stories.md

```diff
@@ -0,0 +1,118 @@
+---
+navTitle: Customer Stories
+meta:
+    title: Marketing - Customer Stories
```

Comment:
Why both? And maybe you should work with @joepavitt to understand why there's many ways of setting a title?

Non-blocking comment btw